lib/gui/app/pages/main/templates/main.tpl.html

```diff
@@ -70,7 +70,11 @@
               <!-- middleEllipses errors on undefined, therefore fallback to empty string -->
               {{ drive.getDrivesTitle() | middleEllipses:20 }}
             </span>
-            <span class="step-drive step-size">{{ main.selection.getCurrentDrive().size | closestUnit }}</span>
+            <span
+              ng-if="main.selection.getSelectedDevices().length < 2"
```

Comment:
Should this be `.length === 1`, as we also don't need to render the size when no drives are selected?

Reply:
Good point, it's also probably easier to understand the intent with `=== 1`. Done that now.